tests/acceptance/Dockerfile.e2e

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.24-alpine AS builder
+
+WORKDIR /app
+
+# Install make
+RUN apk add --no-cache make
```

Comment:
Instead of testing inside a go container. How about 

1) compile the image in the user space and drop it into the ubuntu container

2) compile it in a go inage and then drop it into a ubuntu image using multiple stage build

2 options up to you which one to use

As a result we probably need to compile the acceptance test suites as a binary as well